down_frab_videos.py

```diff
@@ -1200,8 +1213,14 @@ def parse_args_from_parser(parser):
         raise SystemExit("Could not download list of media files: " + str(e))
 
     try:
-        print(" - Fahrplan from \"" + domain_from_url(selected_event["fahrplan"]) + "\".")
-        fahrplan = fahrplan_data(selected_event["fahrplan"])
+        if ("fahrplan_schedule" in selected_event):
+            fahrplan_url = selected_event["fahrplan_schedule"]
+            is_fahrplan_json = True
+        else:
+            fahrplan_url = selected_event["fahrplan"]
+            is_fahrplan_json = False
+        print(" - Fahrplan from \"" + domain_from_url(fahrplan_url) + "\".")
+        fahrplan = fahrplan_data(fahrplan_url, is_fahrplan_json)
```

Comment:
In light of my suggestions around line 627 I would here only pass the `fahrplan_url` to the `fahrplan_data` object for all events but GPN, but for GPN I would pass two appropriate arguments.